AirCasting/CreateSessionViews/CreateSessionDetailsView.swift

```diff
@@ -47,54 +48,68 @@ struct CreateSessionDetailsView: View {
                 .frame(maxWidth: .infinity, minHeight: geometry.size.height, alignment: .top)
             }
         }
-                .simultaneousGesture(
-        
+        .simultaneousGesture(
             DragGesture(minimumDistance: 2, coordinateSpace: .global)
-                .onChanged({ (_) in
+                .onChanged { _ in
                     UIApplication.shared.sendAction(#selector(UIResponder.resignFirstResponder), to: nil, from: nil, for: nil)
-                })
+                }
         )
     }
 }
 
 private extension CreateSessionDetailsView {
     var continueButton: some View {
         Button(action: {
+            showingAlert = false
             sessionContext.sessionName = sessionName
             sessionContext.sessionTags = sessionTags
             if sessionContext.sessionType == SessionType.fixed {
                 sessionContext.isIndoor = isIndoor
             }
             getAndSaveStartingLocation()
             isConfirmCreatingSessionActive = true
-            #warning("TODO: handle wifi empty password")
-            if wifiSSID != "" && wifiPassword != "" {
+            if wifiSSID != "", wifiPassword != "" {
                 sessionContext.wifiSSID = wifiSSID
                 sessionContext.wifiPassword = wifiPassword
+                showingAlert = false
+            } else if isWiFi, wifiSSID == "", wifiPassword == "" {
+                isConfirmCreatingSessionActive = false
+                showingAlert = true
             }
         }, label: {
             Text("Continue")
                 .frame(maxWidth: .infinity)
         })
-        .buttonStyle(BlueButtonStyle())
-        .background( Group {
-            NavigationLink(
-                destination: ConfirmCreatingSessionView(sessionCreator: sessionCreator,
-                                                        creatingSessionFlowContinues: $creatingSessionFlowContinues,
-                                                        sessionName: sessionName),
-                isActive: $isConfirmCreatingSessionActive,
-                label: {
-                    EmptyView()
-                })
-        })
+            .buttonStyle(BlueButtonStyle())
+            .alert(isPresented: $showingAlert, content: {
+                Alert(title: Text("Wi-Fi credentials are empty "),
+                      message: Text("Do you want to pop up Wi-Fi screen?"),
+                      primaryButton: .default(Text("Show Wi-fi screen")) {
+                          isWifiPopupPresented = true
+                      },
+                      secondaryButton: .default(Text("Cancel")))
+            })
+            .background(Group {
+                if (!isWiFi && wifiSSID == "" && wifiPassword == "") || (!isWiFi && wifiSSID != "" && wifiPassword != "") || (isWiFi && wifiSSID != "" && wifiPassword != "") {
```

Comment:
I think we don't need it here. 
NavigationLink is inactive until `isConfirmCreatingSessionActive = true` and we check the wifi credentials in a button action. 🙂